recipes/openfst/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: openfst
+  version: "1.6.1"
+
+source:
+  fn: openfst-1.6.1.tar.gz
+  url: http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-1.6.1.tar.gz
+  sha1: c623a39394158cc5abd2e54c6dd7934b07394440
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - autoconf
+    - python
+  run:
+    - toolchain
+
+test:
+  requires:
+    - toolchain
```

Comment:
Can you remove it from test for now? If the tests fail we might want to put it back.